crates/indexer/src/accountsdb/accounts/metadata.rs

```diff
@@ -42,16 +47,27 @@ pub(crate) async fn process(client: &Client, key: Pubkey, meta: MetadataAccount)
         .context("Failed to insert metadata")?;
 
     client
-        .dispatch_metadata_json(key, meta.data.uri.trim_end_matches('\0').to_owned())
+        .dispatch_metadata_json(
+            key,
+            first_verified_creator,
+            meta.data.uri.trim_end_matches('\0').to_owned(),
+        )
```

Comment:
Due to the way HTTP indexer jobs are deduplicated currently, this may be an issue and require discussion and/or refactoring.